app/middlewares/uniqueEmail.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../models');
+const errors = require('../errors');
+
+const uniqueEmail = async (req, res, next) => {
```

Comment:
Generally functions should be actions, so the code is easier to read, maybe `checkUniqueEmail` or `validateEmail` or `checkIfEmailExists`, And maybe use a more generic name for the file, like validators or something 👍